libs/stats/odc/stats/_cli_save_tasks.py

```diff
@@ -29,7 +29,9 @@
     help="Only extract datasets for a given time range, Example '2020-05--P1M' month of May 2020",
 )
 @click.option(
-    "--frequency", type=str, help="Specify temporal binning: annual|semiannual|seasonal|all"
+    "--frequency",
+    type=str,
+    help="Specify temporal binning: annual|semiannual|seasonal|all",
```

Comment:
Did you say there was a new option for this?
```suggestion
    help="Specify temporal binning: annual|annual-fy|semiannual|seasonal|all",
```